src/redux/actions/experiments/createExperiment.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-param-reassign */
+import moment from 'moment';
+import hash from 'object-hash';
+
+import saveExperiment from './saveExperiment';
+import {
+  EXPERIMENTS_CREATE,
+  EXPERIMENTS_ERROR,
+} from '../../actionTypes/experiments';
+import { experimentTemplate } from '../../reducers/experiments/initialState';
+import pushNotificationMessage from '../notifications';
+import errorTypes from './errorTypes';
+
+const unnamedExperimentName = 'Unnamed Experiment';
+const defaultDescription = 'Add description here';
```

Comment:
extract this and the function that is assigning default name to an experiment to happen from the react component itself and then in this action only pass the experiment name.